core/src/test/java/org/apache/iceberg/rest/requests/TestCreateTableRequest.java

```diff
@@ -257,7 +255,7 @@ public void testBuilderDoesNotBuildInvalidRequests() {
 
   @Override
   public String[] allFieldsFromSpec() {
-    return new String[] {"name", "location", "schema", "spec", "order", "properties", "stageCreate"};
+    return new String[] {"name", "location", "schema", "spec", "order", "stage-create", "properties"};
```

Comment:
Since we caught that this wasn't correctly testing against the spec with `stageCreate`, I just [checked the spec](https://github.com/apache/iceberg/blob/master/open-api/rest-catalog-open-api.yaml#L1514-L1517) and it looks like there are more incorrect fields: `partition-spec` and `write-order`. Can you also fix those fields?

Reply:
Sure

Reply:
Fixed.